src/crypto/store/localStorage-crypto-store.js

```diff
@@ -369,7 +369,7 @@ export class LocalStorageCryptoStore extends MemoryCryptoStore {
 
     getSecretStorePrivateKey(txn, func, type) {
         const key = getJsonItem(this.store, E2E_PREFIX + `ssss_cache.${type}`);
-        func(key ? Uint8Array.from(key) : key);
```

Comment:
None of the other crypto stores have these conversions, and they break the encrypted storage of the private keys.